layouts/404.html

```diff
@@ -1,4 +1,4 @@
-{{ define "main" }}
+{{- define "main" }}
```

Comment:
Generally all `{{` have been changed to `{{-` to strip preceding whitespace in the output, except for cases mentioned in other comments here.